platforms/web/lib/useListeners/types.ts

```diff
@@ -28,15 +26,17 @@ export type LinkEvent = Omit<InputEvent, 'data'> & {
     data: { url: string; text?: string };
 };
 
-type AnchorElementAttributes =
-    | keyof HTMLAttributes<HTMLAnchorElement>
-    | `data-${string}`;
-
-export type Attributes = {
-    [K in AnchorElementAttributes]?: string;
-};
+export type AllowedMentionAttributes = Map<
+    'style' | 'data-mention-type',
+    string
+>;
```

Comment:
This change the type to be a `Map`, which fixes another open issue and we also make it less permissive in terms of what it accepts.